apps/user_status/src/views/Dashboard.vue

```diff
@@ -0,0 +1,97 @@
+<!--
+  - @copyright Copyright (c) 2020 Georg Ehrke <oc.list@georgehrke.com>
+  - @author Georg Ehrke <oc.list@georgehrke.com>
+  -
+  - @license GNU AGPL version 3 or any later version
+  -
+  - This program is free software: you can redistribute it and/or modify
+  - it under the terms of the GNU Affero General Public License as
+  - published by the Free Software Foundation, either version 3 of the
+  - License, or (at your option) any later version.
+  -
+  - This program is distributed in the hope that it will be useful,
+  - but WITHOUT ANY WARRANTY; without even the implied warranty of
+  - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  - GNU Affero General Public License for more details.
+  -
+  - You should have received a copy of the GNU Affero General Public License
+  - along with this program. If not, see <http://www.gnu.org/licenses/>.
+  -
+  -->
+
+<template>
+	<DashboardWidget
+		id="user-status_panel"
+		:items="statuses"
+		:loading="loading">
+		<template v-slot:default="{ item }">
+			<DashboardWidgetItem :item="getWidgetItem(item)">
+				{{ item }}
+			</DashboardWidgetItem>
+		</template>
```

Comment:
You don't need to redefine the default slot for `DashboardWidget` component. It already uses `DashboardWidgetItem`.

```suggestion
	<DashboardWidget
		id="user-status_panel"
		:items="items"
		:loading="loading">
```
with `items` being a computed prop like that:
```
items() {
    return this.statuses.map((s) => {
        return this.getWidgetItem(s)
    })
},
```

Reply:
Excellent point!